addons/arsenal/functions/fnc_statTextStatement_scopeVisionMode.sqf

```diff
@@ -0,0 +1,68 @@
+#include "script_component.hpp"
+/*
+ * Author: Dedmen
+ * Text statement for the scope Night vision support stat.
+ *
+ * Arguments:
+ * 0: not used
+ * 1: item config path (CONFIG)
+ *
+ * Return Value:
+ * String to display
+ *
+ * Public: No
+ */
+
+params ["", "_config"];
+TRACE_1("statTextStatement_scopeVisionMode",_config);
+
+private _opticsModes = ("true" ConfigClasses (_config >> "ItemInfo" >> "OpticsModes")) apply {
```

Comment:
```suggestion
private _opticsModes = ("true" configClasses (_config >> "ItemInfo" >> "OpticsModes")) apply {
```

Reply:
Wow I "fixed" from configclasses to ConfigClasses :man_facepalming: